gslab_scons/size_warning.py

```diff
@@ -128,3 +184,30 @@ def create_size_dictionary(dirs):
 
     return size_dictionary
 
+def add_to_lfs(filepath, attrib_path = '../.gitattributes'):
+    '''
+    This function tracks existing files with git lfs by writing down 
+    the given file path to the git attribute file (the default path is '../.gitattributes')
+    '''
+    with open(attrib_path, "a") as f:
+        f.write('\n%s filter=lfs diff=lfs merge=lfs -text' % filepath)
+    return None
```

Comment:
A file larger than 2M and not in `.gitattributes` will trigger the warnings twice. Do you think this might be a problem? I'm fine with it.
![image](https://user-images.githubusercontent.com/16677008/29900331-10603bc8-8da6-11e7-9ecd-5af416b35684.png)

Reply:
I'm fine with it as well.